src/sonic-yang-models/yang-models/sonic-system-ldap.yang

```diff
@@ -0,0 +1,128 @@
+module sonic-system-ldap {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-system-ldap";
+    prefix ssys;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-port {

```

Comment:
Please remove unused imports such as port, lag, loopback, interface and mgmt-port.

Reply:
done